THIRD-PARTY-LICENSES.csv

```diff
@@ -485,102 +448,94 @@
 "dom-serialize@2.2.1","MIT","https://github.com/webmodules/dom-serialize"
 "dom-serializer@1.3.2","MIT","https://github.com/cheeriojs/dom-renderer"
 "domelementtype@2.2.0","BSD-2-Clause","https://github.com/fb55/domelementtype"
-"domhandler@4.2.0","BSD-2-Clause","https://github.com/fb55/domhandler"
+"domhandler@4.3.0","BSD-2-Clause","https://github.com/fb55/domhandler"
 "domino@2.1.6","BSD-2-Clause","https://github.com/fgnass/domino"
-"dompurify@2.2.9","(MPL-2.0 OR Apache-2.0)","https://github.com/cure53/DOMPurify"
-"domutils@2.7.0","BSD-2-Clause","https://github.com/fb55/domutils"
+"dompurify@2.3.5","(MPL-2.0 OR Apache-2.0)","https://github.com/cure53/DOMPurify"
+"domutils@2.8.0","BSD-2-Clause","https://github.com/fb55/domutils"
 "duplexify@3.7.1","MIT","https://github.com/mafintosh/duplexify"
 "ecc-jsbn@0.1.2","MIT","https://github.com/quartzjer/ecc-jsbn"
 "ee-first@1.1.1","MIT","https://github.com/jonathanong/ee-first"
-"electron-to-chromium@1.3.752","ISC","https://github.com/kilian/electron-to-chromium"
-"emoji-regex@7.0.3","MIT","https://github.com/mathiasbynens/emoji-regex"
+"ejs@3.1.6","Apache-2.0","https://github.com/mde/ejs"
+"electron-to-chromium@1.4.67","ISC","https://github.com/kilian/electron-to-chromium"
```

Comment:
https://en.wikipedia.org/wiki/ISC_license
Interesting, I should probably have known about this already